presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -85,39 +88,127 @@ public static PlanMatchPattern tableScan(String expectedTableName)
         return node(TableScanNode.class).with(new TableScanMatcher(expectedTableName));
     }
 
-    public static PlanMatchPattern tableScan(String expectedTableName, Map<String, Domain> constraint)
+    public static PlanMatchPattern tableScan(String expectedTableName, Map<String, String> columnReferences)
```

Comment:
Yes, but very few tests actually care about the constraints, so I think providing the convenience method is worthwhile.